Framework/Algorithms/test/RebinTest.h

```diff
@@ -118,6 +118,59 @@ class RebinTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().remove("test_out");
   }
 
+  void test_make_three_params() {
+    // test that if only a binwidth is given, will create an xmin, xmax from xstep
+    Workspace2D_sptr test_in1D = Create1DWorkspace(50);
+    AnalysisDataService::Instance().add("test_in1D", test_in1D);
+
+    Rebin rebin;
+    rebin.initialize();
+    rebin.setPropertyValue("InputWorkspace", "test_in1D");
+    rebin.setPropertyValue("OutputWorkspace", "test_out");
+    rebin.setPropertyValue("Params", "2.0");
+    TS_ASSERT_THROWS_NOTHING(rebin.execute());
+    TS_ASSERT(rebin.isExecuted());
+
+    // ensure that the properties are not changed in default mode
+    std::vector<double> rbParams = rebin.getProperty("Params");
+    TS_ASSERT_EQUALS(rbParams.size(), 1);
+    TS_ASSERT_EQUALS(2.0, rbParams[0]);
+
+    AnalysisDataService::Instance().remove("test_in1D");
+    AnalysisDataService::Instance().remove("test_out");
+  }
+
+  void test_make_three_params_modeset() {
+    // test that if only a binwidth is given, will create an xmin, xmax from xstep
```

Comment:
this has the same comment as the function above, but I don't think does the same thing?